virtualxt.freedos

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+HERE="$(cd "$(dirname "$0")" && pwd -P)"
+DATA=$HERE/../share/virtualxt
+
+mkdir -p $HOME/.virtualxt/boot
+cp -n $DATA/freedos_hd.img $HOME/.virtualxt/boot/
```

Comment:
why can't this be in `$XDG_DATA_HOME` or another XDG directory instead of writing in `$HOME`

Reply:
I do not have `$XDG_DATA_HOME` defined on my system. (can't seem to find it at least?)
The application is also assuming certain config files in `$HOME/.virtualxt/` so it seems like an unnecessary thing to spread things out to more places. The proper solution would be to fix this upstream I think..?

Reply:
inside the sandbox it will be:

the spec: https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html#variables

>  `$XDG_DATA_HOME` defines the base directory relative to which user-specific data files should be stored. If `$XDG_DATA_HOME` is either not set or empty, a default equal to `$HOME/.local/share` should be used. 

In your case it would be resolving to `$HOME/.var/app/org.virtualxt.VirtualXT/data`

This would also mean changing `VXT_DEFAULT_HD_IMAGE` below

Reply:
Should now use `$XDG_DATA_HOME` instead of `$HOME`.